ui/src/pages/backoffice/Loan/LoanDetails/components/LoanMetadata/LoanMetadata.js

```diff
@@ -1,83 +1,83 @@
 import { invenioConfig } from '@config';
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
-import { Divider, Grid, Header, Message, Segment } from 'semantic-ui-react';
-import { ItemMetadata } from '../ItemMetadata';
-import { LoanActions } from '../LoanActions';
-import { AvailableItems } from '../AvailableItems';
+import { Grid, Icon, Label } from 'semantic-ui-react';
 import {
-  OverdueLoanSendMailModal,
   MetadataTable,
+  PatronDetailsLink,
+  LocationsLink,
+  ItemIcon,
+  DocumentIcon,
+  PatronIcon,
 } from '@pages/backoffice/components';
 import isEmpty from 'lodash/isEmpty';
-import { toShortDateTime } from '@api/date';
+import { toShortDate, toShortDateTime } from '@api/date';
 import { LoanLinkToItem } from '../../../../components/Loan';
+import { DocumentTitle } from '@components/Document';
 
 export default class LoanMetadata extends Component {
-  constructor(props) {
-    super(props);
-    const isRequest = invenioConfig.circulation.loanRequestStates.includes(
-      this.props.loanDetails.metadata.state
-    );
-    this.state = {
-      isAvailableItemsVisible: isRequest,
-    };
-  }
-
-  showAvailableItems = (newState = true) => {
-    this.setState({ isAvailableItemsVisible: newState });
-  };
-
-  isItemDeleted = () => {
-    const data = this.props.loanDetails;
-    return isEmpty(data.metadata.item) && data.metadata.item_pid;
-  };
-
-  renderItem() {
-    const data = this.props.loanDetails;
-    return (
-      !isEmpty(data.metadata.item) && (
-        <ItemMetadata
-          item={data.metadata.item}
-          loanState={data.metadata.state}
-          changeItemClickHandler={this.showAvailableItems}
-        />
-      )
-    );
-  }
-
-  renderDeletedItem() {
-    const data = this.props.loanDetails;
-    return (
-      this.isItemDeleted() && (
-        <Message
-          warning
-          icon="trash alternate"
-          header={`Item deleted`}
-          content={`The item ${data.metadata.item_pid.value} assigned to this loan has been deleted.`}
-        />
-      )
-    );
-  }
-
   prepareLeftData(data) {
-    const rows = [
-      { name: 'Document pid', value: data.metadata.document_pid },
+    return [
       {
-        name: 'Item pid',
+        name: 'State',
+        value: (
+          <Label basic color="blue" size="tiny">
+            {data.metadata.state}
+          </Label>
+        ),
+      },
+      {
+        name: (
+          <>
+            <DocumentIcon />
+            Document
+          </>
+        ),
+        value: <DocumentTitle metadata={data.metadata.document} />,
+      },
+      {
+        name: (
+          <>
+            <ItemIcon />
+            Physical copy
```

Comment:
Sometimes we use/show "Physical copy", sometimes "Item". Shall we align them to one?

Reply:
"Item" is only in the code, because of the data model, physical copy is what we display. I try to change all the display ones to "Physical copy" when I see, but I might miss some
I'm not sure if you want also to change the names in the code?